### PR TITLE
Add and edit users

### LINKED ISSUE #33: Edit users
I should be able to edit users

### PR DESCRIPTION
**Pull request by djensenius**
Closes #33

### CODE REVIEW

File: src/services/admin-api.ts [hunk 1]
```diff
@@ -14,12 +16,13 @@ export interface AdminApi {
 }
 
 export interface User {
-  id: number;
+  id?: number;
```
Comment (MaggieShan) [inline src/services/admin-api.ts:21]: Should these be required, so we are guaranteed to have a user's info?

Comment (djensenius) [reply src/services/admin-api.ts:21]: When creating a new user, there is not yet an ID. So technically it's optional...

https://github.com/condomanagement/condo-web/pull/73/files#diff-69a5cc7b44e72ee82a4e222adc86c224R378

Open to ideas though!

Comment (MaggieShan) [reply src/services/admin-api.ts:21]: Oh okay that's true! 
Hmm I'm not sure about ID, but we can probably add form validation for name/unit/email/phone instead then?

Comment (djensenius) [reply src/services/admin-api.ts:21]: Let's add form validation for these as part of the other story to handle better form validation. Phone is optional though.

**Comment by djensenius** [comment]
@MaggieShan Looks like we need this one approved since the web won't compile without it, preventing me from reviewing your pr 😢

**Review by MaggieShan — APPROVED** [review]
lgtm

**Pull request merged**